src/net/sourceforge/kolmafia/objectpool/ItemPool.java

```diff
@@ -3569,6 +3569,7 @@ public class ItemPool {
   public static final int SHADOW_PILL = 11184;
   public static final int SHADOW_NEEDLE = 11185;
   public static final int CURSED_MONKEY_PAW = 11186;
+  public static final int REPLICA_MR_ACCESSORY = 11189;
```

Comment:
You need to add this to ResultProcessor.gainItem where gaining/losing a currency item signals (coinmasters) to alert the CoinmastersFrame to update.